tests/test_pipenv.py

```diff
@@ -174,6 +174,14 @@ def test_man(self):
             c = p.pipenv('--man')
             assert c.return_code == 0 or c.err
 
+    @pytest.mark.cli
+    @pytest.mark.install
+    def test_install_parse_error(self):
+        with PipenvInstance() as p:
+            c = p.pipenv('install tablib --upgrade')
```

Comment:
I would suggest to put a comment here to explain that the invalid input is `--upgrade` here. The reason of the parse error is not clear at first.
(I'm sure our future selves could forget about that one :) )

Reply:
Yep, a comment is a good call. Then I think we're set to merge here. We'll take care of the option issue in another PR.